frontend/src/metabase/admin/settings/components/EmbeddingSettings/InteractiveEmbeddingSettings.tsx

```diff
@@ -0,0 +1,7 @@
+import RedirectWidget from "../widgets/RedirectWidget";
+
+export function InteractiveEmbeddingSettings() {
+  return (
+    <RedirectWidget to="/admin/settings/embedding-in-other-applications" />
```

Comment:
follow the logic in `selectors.js` we shouldn't be able to visit this page without having a pro license.